bumble/device.py

```diff
@@ -2110,11 +2110,29 @@ async def update_connection_parameters(
         supervision_timeout,
         min_ce_length=0,
         max_ce_length=0,
-    ):
+        use_l2cap=False,
```

Comment:
Would be good to also add this parameter to the `Connection.update_parameters` method that calls this.